app/components/SpeakingTimeContent.tsx

```diff
@@ -47,10 +49,14 @@ export default function SpeakingTimeContent({
   return (
     <div>
       <ContentBlocks content={page.fields.content} locale={locale} />
+
       <details open={true} className="mx-auto max-w-7xl py-8 px-4">
-        <summary>
+        <summary style={{ display: "inline-flex", cursor: "pointer" }}>
+          <FilterIcon />
           <h5 className="inline-block text-xl">{t("filter.showFilter")}</h5>
+          <ArrowFilterIcon />
```

Comment:
The arrow doesn't turn when clicking on the filter icon or the 'filter anzeigen'. Maybe the condition checking if it's expanded or not should come from this component and leave the children component with little logic as possible - it could be something like (pseudocode:)
```
if (<details> open is true)
<ArrowUp> --> only containing the svg
else
<ArrowDown> --> only containing the svg
```